docs/api.md

```diff
@@ -534,10 +581,10 @@ Access-Control-Expose-Headers: Location
 ### Access by Key
 Checks if thing has access to a channel
 
-> Must-have: `channel_id` and `thing_key`
+> Must-have: `thing_key`
 
 ```bash
-curl -s -S -i -X POST -H "Content-Type: application/json" http://localhost/identify/channels/<channel_id>/access-by-key -d '{"token": "<thing_key>"}'
+curl -s -S -i -X POST -H "Content-Type: application/json" http://localhost/connections -d '{"token": "<thing_key>"}'
```

Comment:
This is wrong, token is never in the request body but in the Auth header. Add `-H "Authorization: Thing <thing_key>"`

Reply:
Instead of `token` should be `key` in the request body, https://github.com/MainfluxLabs/mainflux/blob/master/things/api/http/transport.go#L675